packages/app-layout/src/AppLayoutContent.tsx

```diff
@@ -0,0 +1,125 @@
+import { Fragment, PropsWithChildren } from 'react';
+import { Prose } from '@ag.ds-next/react/prose';
+import { PageContent } from '@ag.ds-next/react/content';
+import { Callout } from '@ag.ds-next/react/callout';
+import {
+	boxPalette,
+	CoreProvider,
+	mapSpacing,
+	Spacing,
+	tokens,
+} from '@ag.ds-next/react/core';
+import { Text } from '@ag.ds-next/react/text';
+import { TextLink } from '@ag.ds-next/react/text-link';
+
+export type ExpectedClaims = {
+	name?: string;
+	given_name?: string;
+	family_name?: string;
+
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	[others: string]: any;
+};
+
+const isEmptyString = (t: string | undefined) => (t?.trim() ?? '') === '';
+
+const Divider = (props: { spacing?: Spacing }) => (
+	<hr
+		css={{
+			boxSizing: 'content-box',
+			height: 0,
+			overflow: 'visible',
+			border: 'none',
+			borderTopWidth: tokens.borderWidth.sm,
+			borderTopStyle: 'solid',
+			borderColor: boxPalette.borderMuted,
+			width: '100%',
+			marginTop: mapSpacing(props?.spacing ?? 2),
+			marginBottom: mapSpacing(props?.spacing ?? 2),
```

Comment:
These don't feel necessary (or used) so could you use the DS' `Divider` component?